types/lolex/index.d.ts

```diff
@@ -167,23 +174,42 @@ export interface LolexClock<TTimerId extends TimerId> {
  */
 export declare function createClock<TClock extends Clock>(now?: number | Date, loopLimit?: number): TClock;
 
-/**
- * Creates a clock and installs it globally.
- * 
- * @param now   Current time for the clock, as with lolex.createClock().
- * @param toFake   Names of methods that should be faked.
- * @type TClock   Type of clock to create.
- * @usage lolex.install(["setTimeout", "clearTimeout"]);
- */
-export declare function install<TClock extends Clock>(now?: number | Date, toFake?: FakeMethod[]): TClock;
+
+export interface LolexInstallOpts {
+    /**
+     * installs lolex onto the specified target context (default: global)
```

Comment:
Nit: in the existing jsdoc comments, everything starts with with a capital letter (as per the comment-format TSLint rule's `check-uppercase` option) and ends with a period. It would be cleaner to adhere to that style of comments.

Reply:
fixed